java-frontend/src/main/java/org/sonar/java/Measurer.java

```diff
@@ -182,8 +180,8 @@ private static boolean isClassTree(Tree tree) {
     return tree.is(Tree.Kind.CLASS) || tree.is(Tree.Kind.INTERFACE) || tree.is(Tree.Kind.ENUM) || tree.is(Tree.Kind.ANNOTATION_TYPE);
   }
 
-  private void saveMetricOnFile(Metric metric, double value) {
-    sensorContext.saveMeasure(sonarFile, new Measure(metric, value));
+  private <T extends Serializable> void saveMetricOnFile(Metric<T> metric, T value) {
+    sensorContext.<T>newMeasure().forMetric(metric).on(sonarFile).withValue(value).save();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)